package.json

```diff
@@ -14,58 +14,58 @@
-    "@angular/core": "15.0.4",
-    "@angular/forms": "15.0.4",
-    "@angular/platform-browser": "15.0.4",
-    "@angular/platform-browser-dynamic": "15.0.4",
-    "@angular/router": "15.0.3",
-    "@ngx-translate/core": "14.0.0",
-    "@ngx-translate/http-loader": "^7.0.0",
+    "@angular/animations": "16.2.7",
+    "@angular/cdk": "16.2.7",
+    "@angular/common": "16.2.7",
+    "@angular/compiler": "16.2.7",
+    "@angular/core": "16.2.7",
+    "@angular/forms": "16.2.7",
+    "@angular/platform-browser": "16.2.7",
+    "@angular/platform-browser-dynamic": "16.2.7",
+    "@angular/router": "16.2.7",
+    "@ngx-translate/core": "15.0.0",
+    "@ngx-translate/http-loader": "8.0.0",
     "ag-grid-angular": "^28.2.1",
     "ag-grid-community": "^28.2.1",
-    "angular-split": "^4.0.0",
+    "angular-split": "16.2.1",
     "bootstrap": "^4.5.2",
-    "core-js": "^2.5.6",
-    "date-fns": "^2.15.0",
-    "interactjs": "^1.10.17",
+    "core-js": "3.32.2",
+    "date-fns": "2.30.0",
+    "interactjs": "^1.10.19",
     "jquery": "^3.5.1",
     "mobile-drag-drop": "^2.3.0-rc.2",
-    "pako": "^1.0.6",
+    "pako": "^1.0.11",
     "popper.js": "^1.16.1",
     "primeicons": "^6.0.1",
     "primeng": "^14.1.2",
-    "rxjs": "^7.5.7",
-    "systelab-preferences": "~15.0.0",
-    "systelab-translate": "~15.0.0",
+    "rxjs": "7.8.1",
+    "systelab-preferences": "~16.0.1",
+    "systelab-translate": "~16.0.1",
     "tslib": "^2.1.0",
-    "zone.js": "~0.11.4"
+    "zone.js": "0.13.3"
   },
   "devDependencies": {
+    "@angular-devkit/build-angular": "16.2.6",
+    "@angular-eslint/builder": "16.2.0",
+    "@angular-eslint/eslint-plugin": "16.2.0",
+    "@angular-eslint/eslint-plugin-template": "16.2.0",
+    "@angular-eslint/schematics": "16.2.0",
+    "@angular-eslint/template-parser": "16.2.0",
+    "@angular/cli": "16.2.6",
+    "@angular/compiler-cli": "16.2.7",
+    "@angular/language-service": "16.2.7",
     "@babel/core": "^7.20.2",
     "@babel/preset-env": "^7.20.2",
     "@babel/register": "^7.18.9",
-    "@angular-devkit/build-angular": "^15.0.4",
-    "@angular-eslint/builder": "^15.1.0",
-    "@angular-eslint/eslint-plugin": "^15.1.0",
-    "@angular-eslint/eslint-plugin-template": "^15.1.0",
-    "@angular-eslint/schematics": "^15.1.0",
-    "@angular-eslint/template-parser": "^15.1.0",
-    "@angular/cli": "^15.0.4",
-    "@angular/compiler-cli": "15.0.4",
-    "@angular/language-service": "15.0.4",
     "@types/chart.js": "^2.9.24",
     "@types/jasmine": "~3.5.0",
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "^18.11.9",
     "@typescript-eslint/eslint-plugin": "^5.36.2",
     "@typescript-eslint/parser": "^5.36.2",
-    "eslint": "^8.23.0",
-    "eslint-plugin-import": "2.25.2",
-    "eslint-plugin-jsdoc": "39.3.6",
+    "eslint": "8.51.0",
+    "eslint-plugin-import": "2.28.1",
+    "eslint-plugin-jsdoc": "46.8.2",
     "eslint-plugin-prefer-arrow": "1.2.3",
     "gulp": "^4.0.2",
     "jasmine-core": "~3.5.0",
```

Comment:
The last validated jasmine version was 3.8.0 with karma 6.3.8.
As of today in our product, Jasmine is 3.10.2 and karma v6.4.1 and we need to update these dependencies because additional OTS is required. Please, consider to update them to the latest.